projects/ngrx.io/content/guide/entity/install.md

```diff
@@ -0,0 +1,31 @@
+# Installation
+
+## Installing with `npm`
+
+For more information on using `npm` check out the docs <a href="https://docs.npmjs.com/cli/install" target="_blank">here</a>.
+
+```sh
+npm install @ngrx/entity --save
+```
+
+## Installing with `yarn`
+
+For more information on using `yarn` check out the docs <a href="https://yarnpkg.com/docs/usage" target="_blank">here</a>.
+
+```sh
+yarn add @ngrx/entity
+```
+
+## Installing with `ng add`
+
+If your project is using the Angular CLI 6+ then you can install the Entity to your project with the following `ng add` command <a href="https://angular.io/cli/add" target="_blank">(details here)</a>:
+
+```sh
+ng add @ngrx/entity
+```
+
+This command will automate the following steps:
+
+1. Update `package.json` > `depedencies` with `@ngrx/entity`.
```

Comment:
```suggestion
1. Update `package.json` > `dependencies` with `@ngrx/entity`.
```

Reply:
This should be corrected in latest commit